ui-core/src/components/inputs/ComboBox/ComboBox.tsx

```diff
@@ -198,8 +198,7 @@ const ComboBox = <T,>({
     } else if (!isInputInSuggestions && selectedOption) {
       setInputValue(getSuggestionLabel(selectedOption));
     } else if (!isInputInSuggestions) {
-      setInputValue('');
-      setSelectedOption(null);
+      clearInput();
```

Comment:
`clearInput()` will `focusInput()` at the end. This is confusing for users: they've navigated away from the input (either by clicking outside, or with `Tab`, or something else), but then focus is set back to the input.